Changelog

```diff
@@ -1,3 +1,5 @@
+- Fix: bad json measures are reported as 500 Error instead of 400 (#863)
+
```

Comment:
```suggestion
```

After LGTM I have realized this should be to CHANGES_NEXT_RELEASE...

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/864/commits/d6fd25d1c8779e06f4c2094fb7afaf5b3fbf9207